docker/docker.go

```diff
@@ -84,12 +84,6 @@ func main() {
 
 	setDefaultConfFlag(flTrustKey, defaultTrustKeyFile)
 
-	// Regardless of whether the user sets it to true or false, if they
-	// specify --tlsverify at all then we need to turn on tls
-	if flag.IsSet("-tlsverify") {
```

Comment:
correct.  The behavior that code is trying to do is: if they specify "--tlsverify=false" then they ant TLS enabled but not tlsverify.  IOW, --tlsverify=false should imply --tls